src/server.h

```diff
@@ -2132,7 +2138,9 @@ void updateDictResizePolicy(void);
 int htNeedsResize(dict *dict);
 void populateCommandTable(void);
 void resetCommandTableStats(void);
+void resetErrorTableStats(void);
 void adjustOpenFilesLimit(void);
+void incrementError(const char* fullerr, size_t namelen);
```

Comment:
style: * typically sticks with the variable name in Redis.

Reply:
renamed to `incrementErrorCount`

Reply:
@madolson now fixed :)